src/null-safety/understanding-null-safety.md

```diff
@@ -755,10 +757,10 @@ We've now corralled `null` into the set of nullable types. With flow analysis,
 we can safely let some non-`null` values hop over the fence to the non-nullable
 side where we can use them. That's a big step, but if we stop here, the
 resulting system is still painfully restrictive. Flow analysis only helps with
-locals and parameters.
+locals, parameters, and private, final fields.
```

Comment:
This is an example of a sentence that I think is harder to follow due to the comma between "private" and "final". We're trying to say that flow analysis helps with 3 things, but because of the extra comma, my brain initially interprets it as 4 things, one of which is "private", which isn't a thing. Then I notice that the "and" is in the wrong place. And then finally I figure out it.

Reply:
I agree now that you've pointed it out. I'll try to fix this in all the new PRs so don't worry about commenting every time you see it (it's probably a lot)